wolfcrypt/src/port/st/stsafe.c

```diff
@@ -170,7 +170,7 @@ int SSL_STSAFE_VerifyPeerCertCb(WOLFSSL* ssl,
         #ifdef USE_STSAFE_VERBOSE
             STSAFE_INTERFACE_PRINTF("stsafe_interface_verify error: %d\n", err);
         #endif
-            err = WC_HW_E;
+            err = -err;
```

Comment:
These callbacks are returning values back to wolfSSL, and it doesn't know anything about STSAFE error codes. All the other callbacks return WC_HW_E in this case, as well.

Reply:
This is inside a PK callback for STSAFE specifically and I believe the customer was making these suggestions to enable seeing the error code from the STSAFE layer. I am okay with this change.

Reply:
It is a callback put in the WOLFSSL_CTX. It'll be returned to EccVerify(). Admittedly, it only checks if the return code is non-zero, and if it is reassigns it to VERIFY_SIGN_ERROR. This is a _macht nichts_.